recipes/hictk/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "hictk" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "b88dd0dd798c59d86ababcc03ae466c2a21175afaaee5a837555a3ba0a20b14f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  # url: https://github.com/paulsengroup/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  url: https://github.com/paulsengroup/hictk/archive/refs/heads/fix-conda.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.25
+    - conan >=2.0
+    - git >2.7
```

Comment:
git should not be required during build as all source code should be contained in the package.

Reply:
Good catch!